chain/consensus/mir/pool/fifo/fifo_pool.go

```diff
@@ -0,0 +1,49 @@
+package fifo
```

Comment:
What is the difference between `chain/consensus/mir/pool` and `chain/consensus/mir/mempool`?

Reply:
@dnkolegov, can you take this one?

Reply:
Removed